Sources/Orbit/Components/Collapse.swift

```diff
@@ -0,0 +1,183 @@
+import SwiftUI
+
+/// Hides long or complex information.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/collapse/)
+public struct Collapse<Header: View, Content: View>: View {
+
+    struct _Collapse: View {
```

Comment:
The article does not seem to need the internal structure. Do we have a specific need for it?